functions/consumer/xmpp-consumer/src/test/java/org/springframework/cloud/fn/consumer/xmpp/XmppConsumerConfigurationTests.java

```diff
@@ -0,0 +1,57 @@
+package org.springframework.cloud.fn.consumer.xmpp;
+
+import java.time.Duration;
+import java.util.function.Consumer;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.fn.common.xmpp.XmppConnectionFactoryConfiguration;
+import org.springframework.context.annotation.Import;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.test.annotation.DirtiesContext;
+
+import static org.awaitility.Awaitility.await;
+
+/**
+ * @author Daniel Frey
+ *
+ * @since 4.0.0
+ */
+@SpringBootTest(
+        properties = {
+                "xmpp.factory.user=dmfrey",
+                "xmpp.factory.password=thisisatest",
+                "xmpp.factory.host=jabb3r.org",
```

Comment:
I don't think this is OK to accept this real host in our tests...

Reply:
these will be removed. it was just to test against a real server. Gonna try to come up with a docker version that can be loaded with test containers